code/modules/fabrication/designs/protolathe/designs_misc.dm

```diff
@@ -0,0 +1,54 @@
+/datum/fabricator_recipe/protolathe/misc
+	category = "Prototypes"
+	path = /obj/item/radio/beacon
+
+/datum/fabricator_recipe/protolathe/misc/gps
+	path = /obj/item/gps
+
+/datum/fabricator_recipe/protolathe/misc/beacon_locator
+	path = /obj/item/pinpointer/radio
+
+/datum/fabricator_recipe/protolathe/misc/ano_scanner
+	path = /obj/item/ano_scanner
+
+/datum/fabricator_recipe/protolathe/misc/bag_holding
+	path = /obj/item/storage/backpack/holding
+
+/datum/fabricator_recipe/protolathe/misc/dufflebag_holding
+	path = /obj/item/storage/backpack/holding/duffle
+
+/datum/fabricator_recipe/protolathe/misc/beaker
+	path = /obj/item/chems/glass/beaker/noreact
+
+/datum/fabricator_recipe/protolathe/beaker/get_product_name()
```

Comment:
Should be `/datum/fabricator_recipe/protolathe/misc/beaker`? That's where both bluespace and noreact live